src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnosticsForSpan.cs

```diff
@@ -369,13 +369,24 @@ public async Task<bool> TryGetAsync(ArrayBuilder<DiagnosticData> list, Cancellat
                 var document = executor.AnalysisScope.TextDocument;
 
                 using (_addOperationScope?.Invoke(analyzerTypeName))
+                using (Logger.LogBlock(FunctionId.DiagnosticAnalyzerService_GetDiagnosticsForSpanAsync, KeyValueLogMessage.Create(LogType.UserAction, m => CreateLogProperties(m, analyzer)), cancellationToken))
                 using (_addOperationScope is object ? RoslynEventSource.LogInformationalBlock(FunctionId.DiagnosticAnalyzerService_GetDiagnosticsForSpanAsync, analyzerTypeName, cancellationToken) : default)
                 {
                     var diagnostics = await executor.ComputeDiagnosticsAsync(analyzer, cancellationToken).ConfigureAwait(false);
```

Comment:
so i see ComputeDiagnosticsAsync called in a few places.  i have no idea if this using block is sufficient, or if we're missing cases that are important.

Reply:
This part is removed because we have Manish's great PR to get the data for that https://github.com/dotnet/roslyn/pull/65212